extensions-core/azure-extensions/src/test/java/org/apache/druid/storage/azure/AzureDataSegmentPusherTest.java

```diff
@@ -315,39 +310,23 @@ public void getPathForHadoopWithPrefixTest()
   {
     AzureDataSegmentPusher pusher = new AzureDataSegmentPusher(azureStorage, azureAccountConfig, segmentConfigWithPrefix);
     String hadoopPath = pusher.getPathForHadoop();
-    Assert.assertEquals("wasbs://container@account.blob.core.windows.net/prefix/", hadoopPath);
+    assertEquals("wasbs://container@account.blob.core.windows.net/prefix/", hadoopPath);
   }
 
   @Test
   public void getPathForHadoopWithoutPrefixTest()
   {
     AzureDataSegmentPusher pusher = new AzureDataSegmentPusher(azureStorage, azureAccountConfig, segmentConfigWithoutPrefix);
     String hadoopPath = pusher.getPathForHadoop();
-    Assert.assertEquals("wasbs://container@account.blob.core.windows.net/", hadoopPath);
-  }
-
-  @Test
-  public void test_getPathForHadoop_noArgsWithoutPrefix_succeeds()
-  {
-    AzureDataSegmentPusher pusher = new AzureDataSegmentPusher(azureStorage, azureAccountConfig, segmentConfigWithoutPrefix);
-    String hadoopPath = pusher.getPathForHadoop("");
```

Comment:
`getPathForHadoop(String)` is deprecated, and `AzureDataSegmentPusher` calls `getPathForHadoop()` either way.